### ISSUE TITLE
Formatting and display descriptions of guides

### ISSUE THREAD

**Issue by mutolisp** [inaturalist/INaturalistIOS#429]
Hi,

When I tapped the detail button of the guide, part of the description of tags were truncated by the margin. The figure is as following:
![img_4009](https://user-images.githubusercontent.com/122386/38455594-061eb392-3aad-11e8-96aa-bfeeaa51bd0f.PNG)

thanks!

**Comment by tiwane** [comment]
Yeah, I know guides aren't a priority, but if it's an easy fix, making the formatting work here would be great. 
![image](https://user-images.githubusercontent.com/15896181/62329511-f61adf00-b46a-11e9-8363-207402f681c5.png)
